packages/core/src/components/tag/_tag-group.scss

```diff
@@ -0,0 +1,24 @@
+// Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the terms of the LICENSE file distributed with this project.
+
+@import "../../common/variables";
+@import "./common";
+
+$tag-group-spacing: 1px;
+
+.#{$ns}-tag-group {
+  display: inline-flex;
+
+  .#{$ns}-tag {
+    &:not(:first-child) {
```

Comment:
😢 we're also going to have to handle cases where a tag is wrapped in a `Popover`, just like in button groups.